data/common.yaml

```diff
@@ -1,6 +1,51 @@
+
+  dhouse:
+    - ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQDzNCcmwepH5o/hzYz3B5l60dRKE+RaYojhXOgncnbHqg7ZwBMUScXq8nqV6ro5Vo263KcYTzlxB0GOMxELnRBtvAifID0QjtGj7eLMQ0aY7Mn3IAMUBoDhgEUyBClJ2tTrr62Do4raalsurbu7Z1Xb9k+eV2xud5vdgmwY6TcIhTIdIWNSs5dTTa2ZAV1ntgKQYkD9T0VYx0Jrtj0kabeSm2N/cF2BlqswoR8cwTgsLox4/0dbVlA1eNpMNqrJ91DzQeqcNYoQsw4JiVwJNjxLTdnHbm67YriaXC9JxY8XmSiJbHpBDQHofFm+y9Nfgj1Ey+hQwdQ+5pzQADOJ5WY0xsSkH2TGZVzgPvFBzMlRKdbU6lihpQwK+VqE2pNUE3z9K3zwWgfDr5YLS8WzDAzxjCsaPMuNN5O1wpStGxNVTSYpmhX/h4Nl3z3Ya8yq24rQq6h9MG87O2ZS+Zb2/VgmCi6zgp/Z9/GAZ8p0+fq3EkVkBuW53TW2e6hIjeXbPFO9ijlDwKklNT224C0/m4ZkXr1SXJrrUjCXQ0JpJ+nTUkXRwg4w7gXdF3LJ13QXic4GJrNPIAAgTGPUA8+66iDia9zlIMX8d5GuNlssUll/zGbuGRRRGElqUNMGgrDqmQJLjStttQHuRfhuh6vPD4XQtRQfMrdeqlaXbDOeITdlGw== dhouse-mozilla
+    - ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQDJjwS6Yt0OWvQuN7o+FD7Rw2rJhnVZ0fiknr0ovw2ytMBl9xzOGeOxq+o1WGoyHIhLx2bYUb29518Nnit1mpXn9D5bj7cMlfLnXEzjLS8QkUvCqhxZPOjPn+IRZU6C2dSWu0QcIqhjvhu7ab55ED+NjZVZNovQgmBndQAh2+gWUP1cXS6utTbwvLYSsAPflfLJLnnt1Gs5Ae4B96utTzNLL5XbIdkv5PMziJWtXSXI66EWo0On/zLH6JQ71vL+r8I+l5GWCq2IOihSKRz3/OJ5O/IA3mdN9RKZ0pZMoxTgHhxmBpJ8AGju08K9iViNHCGSbXSvjSIFrbn8l4gs2Fl37nzDJ2zpaib5yhqVVucKmceYFFdulHsb3gpfJIkTb8FVQ43NF8Grj40lfPo2GmEopJntMFCFrLnABzWXKJllKO5UdekQgSs6/P6VBTxmmAhEgMc3T5OsCbc3cb4WHWzQL8NPBFKFsdV2LDiG00wkvFF+WYpyKSykWtYhiSADmDe3Hvjn/xOtqGVsoog5XGYgcbS98cKmJkyMHRx/Mc46QNu2f8WcMmLS9mn4jzOMny2cGDh7P6aOkl2Ym+RMjB/4PEqjBqmcX3mFZzIn18yG6EHraScGgPmMWrPnv8nvK3MHa2fLEF9MA6GuFVTT8Lk3kwkbjsi/0dLOcd0QOKbayw== dhousegeorge_2018-06-08
+    - ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQCfeLZeQj7V+K5XZA0YzxDHJiCt4iUvd6bkLFZR02fkVGsFt3639K955ycwspSoEDgqBHfO4UCv2cqBLQqI7EdYl0FcllVPeKL64XKvZA0cXqlzWmqfrnh9yw78Qw0XnYXTbTXVJDxYZC36VZaNFxUc28lr9r3BfMyrVJZ8v7Lz/2S4kYXtOgrqCbu/K2RiNiJ4uO0mAWivF+NsuWInlrH97YdohE/7kydslv0wGVf1rU1tQ0CYXNQBqir89+0Glshix2TtRdXn6HhNbFNFkkbja246fHt26W6COoF8t2R7Kh/Yvs0FLWLZe8xNWVmca2F+RDwpN9muydfVa0j+9iJu+GXBIsNbivqclcqhGHzTItM1GpGe9wgbugKtOMKQ02+OSGHmyAQhG4Fi675QG7yAz3gDH0DYiai/s032mMyqiOSH4JWFvq6dN32l/cbZMYfB/VMBJwbI0EJOb55Kqn6UDyYCaFZpfJq/w1gXEmpu7XtzuBbsFBum6c/248cGR+RtJ5N0EF098+3IRbDOq4t8cKcdy/rOu2O/iCIIfr3coBunFCUPVw6/r8I0qrFcLjCCKKjEBIpTrCv5ch1uj9P0BqJw+LWsalQsJeVQ/U71tHoUWqINrGXyjvVhHcTbeE22bALTq0iEqwZEPrZCusb5yqpow89zw+MqrT9SCz/S0w== dhouse.house
```

Comment:
Is this temporary?

Reply:
It is permanent until we come up with a way to automate pulling the data from IT.  EIS is working on an API for providing that data from ldap.